### DISCUSSION TITLE
Custom prefix per x-data?

### DISCUSSION THREAD

**Discussion by cipriano200** [alpinejs/alpine#4122] [1. Help]
Lets say there will be 2 wordpress plugins on same page using alpine, and want to use custom prefix:
```
	document.addEventListener('alpine:init', () => {
		Alpine.prefix("sx-");
	})		
```

That will break other plugin.

Another issue i have seen, if "x-ref" exists on a page and is not within x-data, some errors pops

**Comment by cipriano200** [comment]
Found other anwers related to this issue

**Comment by ekwoka** [comment]
You would need to not use the Alpine CDN. Or at least not use two Alpine CDN.

Why would you want to use 2 different Alpine instances?
